hazelcast/src/main/java/com/hazelcast/config/ClassFilterList.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import static java.util.Collections.unmodifiableSet;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Class which holds blacklist and whitelist configuration for {@link JavaSerializationFilterConfig}.
```

Comment:
You don't need to add 'class' since that applies to almost anything.

Why not call this class 'ClassFilter' ? That it is a list isn't relevant to the usage. Actually it isn't a list at all since the content is stored in sets.